src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    const type = actions[i].type;
+
+    if (type === 'addProperties') {
```

Comment:
try use switch case when you want to check a value for multiple possible results